miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt

```diff
@@ -58,6 +70,32 @@ class MiniAppFileChooserDefault(var requestCode: Int) : MiniAppFileChooser {
         return true
     }
 
+    /**
+     * Validation Utility for MimeTypes.
+     * @param mimeTypes The Mimetypes needs to convert to valid types.
+     */
+    @VisibleForTesting
+    internal fun extractValidMimeTypes(mimeTypes: Array<String>): List<String> {
+        val results: MutableList<String> = ArrayList()
+
+        val mtm = MimeTypeMap.getSingleton()
+        for (mime in mimeTypes) {
+            if (mime.trim { it <= ' ' }.startsWith(".")) {
```

Comment:
Isn't a single space the default character for `trim`? So couldn't this be just `mime.trim()`?

Also, you're calling `mime.trim` several times so I think you should either set this to a variable or use something like `mime.trim().let { ... }`